clients/admin-ui/package.json

```diff
@@ -22,6 +22,7 @@
     "lint:fix": "eslint . --fix --ext .ts,.tsx",
     "lint-staged:fix": "lint-staged --diff=main",
     "openapi:generate": "openapi --input http://localhost:8080/openapi.json --output ./src/types/api --exportCore false --exportServices false --indent 2",
+    "openapi:generate-dictionary": "openapi --input http://localhost:8081/openapi.json --output ./src/types/dictionary-api --exportCore false --exportServices false --indent 2",
```

Comment:
I meant to add this here https://github.com/ethyca/fides/pull/4254 but ended up only documenting it without actually adding the command 🤦‍♀️